packages/http/CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # @openfn/language-http
 
+## 6.0.0
+
+### Major Changes
+
+- f741086: The axios library has been removed and the operation API simplified.
+
+  ### New features
+
+  - Error mapping. Provide custom errors to response status codes, eg, Eg:
+    `{errors: {404: "You don't have access"}}`
+  - parseAs option. Overrides the default response type. Eg: `{parseAs: "json"}`
+  - Request logging. All requests are now logged to stdout.
+  - tls options. Pass SSL certificates using the `tls` option on any request.
+
+  ### Breaking changes
```

Comment:
More migration notes with @mtuchi

* The `params` and `qs` parameters have been removed. Use`query` instead (it's the same)
* `keepCookie`, which basically just pulls cookies out of the header and writes them tostate.data. But not in a nice way IMO. We've removed support for this. We can reimplement later, with a spec, if anyone needs it.
* `successCodes` is now the error map.
* `transformResponse` is just a callback? We've removed it. Use the callback isntead.
* agentOptions - use the TLS option. `rejectUnauthorised` might still work? It's unclear
* `reponseType` is `parseAs`
* `authorisation` we do this for you in config. If you need to set it explicitly, she the header  (use the helper)

Reply:
`data` is now `body`